### PR TITLE
Parser created for iosxe show snmp group

### PR DESCRIPTION
**Pull request by qduk**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_snmp.py [hunk 1]
```diff
@@ -79,3 +79,153 @@ def cli(self, output=None):
...
+        p_r_wnone = re.compile(r"^readview :\s+(?P<read>\S+)\s+writeview:\s+<no writeview specified>$")
+        p_rw = re.compile(r"^readview :\s+(?P<read>\S+)\s+writeview:\s+(?P<write>\S+)$")
+
+        # notifyview: <no notifyview specified>
+        p_notify_none = re.compile(r"^notifyview: <no notifyview specified>$")
+        p_notify = re.compile(r"^notifyview:\s+(?P<notify>\S+)$")
+
+        # row status: active	access-list: 90
+        p_row = re.compile(r"^row\s+status:\s+active$")
+        p_row_ac = re.compile(r"^row\s+status:\s+active\s+access-list:\s+(?P<access>\S+)$")
```
Comment (ahersi2-cisco) [inline src/genie/libs/parser/iosxe/show_snmp.py:161]: for each regex pattern, can you put the corresponding output above. If `p_row_ac ` captures dat aslightly different, please place it above. Same for above patterns.

**Review by ahersi2-cisco — CHANGES_REQUESTED** [review]

**Comment by ahersi2-cisco** [comment]
can you please run the unit tests and provide a screen shot? Thank you.

File: src/genie/libs/parser/iosxe/show_snmp.py (hunk 1 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/show_snmp.py:170]: Added.

**Comment by qduk** [comment]
show_snmp_unit

**Review by ahersi2-cisco — APPROVED** [review]

**Review by ThomasJRyan — APPROVED** [review]

**Pull request merged**